static/js/click_handlers.js

```diff
@@ -915,6 +915,10 @@ exports.initialize = function () {
         e.stopPropagation();
     });
 
+    $('a.dropdown-toggle, .dropdown-menu a').on('click', function () {
+        popovers.hide_all();
```

Comment:
Did you test this locally? It looks like you're closing the dropdown menu here because of the changes in the other commit.

Reply:
Yes. It solved the issue. I did not think this would cause problems with the login test.

Reply:
This selector seems overly broad to me.  More discussion here: https://chat.zulip.org/#narrow/stream/9-issues/topic/User.20info.20popover.20does.20not.20hide.20.20on.20opening.20menu/near/843659